misc/cspell.json

```diff
@@ -110,7 +110,128 @@
     "validators",
     "Vertx",
     "whitelisted",
-    "whitelisting"
+    "whitelisting",
+    "código",
```

Comment:
I think excluding the Spanish-language file or including a separate dictionary for it (there is probably one) would both be preferable.

Reply:
You are right, thanks. Spanish page excluded.